servers/servertcp.js

```diff
@@ -234,7 +235,8 @@ var ServerTCP = function (vector, options) {
             requestBuffer.writeUInt16LE(crc, requestBuffer.length - 2);
 
             // emit debug data
-            if (modbus.debug) modbus.emit('debug', {action: 'recive', data: requestBuffer});
+            modbusSerialDebug(JSON.stringify({action: 'receive', data: requestBuffer}));
```

Comment:
@biancode hi, now that we use the `debug` module, can we remove the `modbus.emit('debug') ...` ?
if using `debug` fix https://github.com/yaacov/node-modbus-serial/issues/46 then we should drop the 'debug' emits ?